pkg/dataplane/test/streamconsumergroup_test.go

```diff
@@ -393,6 +717,16 @@ func (m *member) ConsumeClaim(session streamconsumergroup.Session, claim streamc
 	return nil
 }
 
+func (m *member) Abort(session streamconsumergroup.Session) error {
+	m.logger.DebugWith("Abort called", "memberID", m.streamConsumerGroupMember.GetID())
+	m.Called(session)
+	m.stop()
+	//expectedStartRecordIndex := m.expectedStartRecordIndex
+	//numberOfRecordToConsume := m.numberOfRecordToConsume
+	//m.start(expectedStartRecordIndex, numberOfRecordToConsume)
+	return nil
```

Comment:
```suggestion
	//expectedStartRecordIndex := m.expectedStartRecordIndex
	//numberOfRecordToConsume := m.numberOfRecordToConsume
	//m.start(expectedStartRecordIndex, numberOfRecordToConsume)
```

are these needed?

Reply:
Not really, because I only test that `Abort` is called and not actually restarts the handler.
Will remove.